dirsrvtests/tests/suites/basic/basic_test.py

```diff
@@ -401,25 +401,74 @@ def test_basic_backup(topology_st, import_example_ldif):
 
     log.info('test_basic_backup: PASSED')
 
-
-def test_basic_db2index(topology_st, import_example_ldif):
+def test_basic_db2index(topology_st):
     """Assert db2index can operate correctly.
 
     :id: 191fc0fd-9722-46b5-a7c3-e8760effe119
 
     :setup: Standalone instance
 
     :steps:
-        1: call db2index
+        1: Call db2index with a single index attribute
+        2: Call db2index with multiple index attributes
+        3: Call db2index with no index attributes
 
     :expectedresults:
-        1: Index succeeds.
+        1: Index succeeds for single index attribute
+        2: Index succeeds for multiple index attributes
+        3: Index succeeds for all backend indexes which have been obtained from dseldif
 
     """
+
+    indexes = []
+
+    # Error log message to confirm a reindex
+    info_message = 'INFO - bdb_db2index - ' + DEFAULT_BENAME + ':' + ' Indexing attribute: '
+
+    log.info('Start the server')
+    topology_st.standalone.start()
+
+    log.info('Offline reindex, stopping the server')
     topology_st.standalone.stop()
-    topology_st.standalone.db2index()
-    topology_st.standalone.db2index(suffixes=[DEFAULT_SUFFIX], attrs=['uid'])
+
+    log.info('Reindex with a single index attribute')
+    topology_st.standalone.db2index(bename=DEFAULT_BENAME, attrs=['uid'])
+    assert topology_st.standalone.searchErrorsLog(info_message + 'uid')
+
```

Comment:
Is there a way to assert here that you DIDNT reinedx the other BE's?

Reply:
We could check the logs for the non existence of the other BE's reindex messages.